packages/dayjs/src/dayjs-utils.ts

```diff
@@ -0,0 +1,292 @@
+import defaultDayjs from "dayjs";
+import customParseFormat from "dayjs/plugin/customParseFormat";
+import advancedFormat from "dayjs/plugin/advancedFormat";
+defaultDayjs.extend(customParseFormat);
```

Comment:
Move extend after imports;
Do we need to extend manually or require user to pass extended instance?

Reply:
I think it makes sense to extend internally (manually) since it an instance-by-instance method and not global. Considering the plugins are part of peerDeps it should be fine.

Will move the extends below.